### PR TITLE
Adding new targets so it can be more easily embedded without cocoa pods

### PR DESCRIPTION
**Pull request by rafaelsimonmaia**
## What
No code changes, only added a few targets to make it easier to integrate without CocoaPods.

## Test 
Compile each Target. They should all succeed.

### CODE REVIEW

File: LDEventSource.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -158,6 +221,9 @@
 				2838332B17A1BFFE00C0141D /* Frameworks */,
 				2838332A17A1BFFE00C0141D /* Products */,
 				44CADEFF622525D649E4BAEF /* Pods */,
+				78D92EB1220BB50400585033 /* DarklyEventSource-WatchOS-Info.plist */,
```
Comment (markpokornycos) [inline LDEventSource.xcodeproj/project.pbxproj]: These 3 info.plist files are absolute referenced to files on your machine. When I open the project they are red. The files are in my file system. All 3 files are copies of the Info.plist in the LDEventSource directory.

I recommend deleting these 3 plists, and instead referencing the original in the new targets by changing the Info.plist reference in the Packaging Build Setting to `$(SRCROOT)/LDEventSource/Info.plist`

File: LDEventSource.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -331,11 +442,65 @@
 			);
 			dependencies = (
 			);
-			name = DarklyEventSource;
+			name = "DarklyEventSource-iOS";
 			productName = "EventSource-iOS";
 			productReference = 69DE1C251E61765000CF816C /* DarklyEventSource.framework */;
 			productType = "com.apple.product-type.framework";
 		};
+		78D92E9C220BB50400585033 /* DarklyEventSource-WatchOS */ = {
```
Comment (markpokornycos) [inline LDEventSource.xcodeproj/project.pbxproj]: #picky name this to `DarklyEventSource-watchOS`

File: LDEventSource.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -331,11 +442,65 @@
...
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = "DarklyEventSource-WatchOS";
+			productName = "EventSource-iOS";
+			productReference = 78D92EB0220BB50400585033 /* DarklyEventSource.framework */;
+			productType = "com.apple.product-type.framework";
+		};
+		78D92EB2220BB51700585033 /* DarklyEventSource-tvOs */ = {
```
Comment (markpokornycos) [inline LDEventSource.xcodeproj/project.pbxproj]: #picky name this to `DarklyEventSource-tvOS`

File: LDEventSource.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -331,11 +442,65 @@
...
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = "DarklyEventSource-tvOs";
+			productName = "EventSource-iOS";
+			productReference = 78D92EC6220BB51700585033 /* DarklyEventSource.framework */;
+			productType = "com.apple.product-type.framework";
+		};
+		78D92EC8220BB51F00585033 /* DarklyEventSource-MacOs */ = {
```
Comment (markpokornycos) [inline LDEventSource.xcodeproj/project.pbxproj]: #picky name this `DarklyEventSource-macOS`

**Review by markpokornycos — CHANGES_REQUESTED** [review]

**Comment by markpokornycos** [comment]
Add schemes for DarklyEventSource_tvOS & _macOS

**Review by markpokornycos — APPROVED** [review]

**Comment by rafaelsimonmaia** [comment]
Looks good now. Thanks Mark.

**Review by arun251 — APPROVED** [review]
👍

**Pull request merged**